Engineering-Reference/M365DestinationAO.md

```diff
@@ -77,87 +77,85 @@ Default: Blank
 
 The access key for the Azure storage account.
 
-Change Conditions: Only avialable if the option SharePoint Migration API is enabled and SharePoint Storage is set to Azure Hosted. 
+Change Conditions: Only available if the option SharePoint Migration API is enabled and SharePoint Storage is set to Azure Hosted. 
 
 ### Retry Backoff (seconds) <a name="sretryback"></a>
 {: .no_toc }
 Default Value: 3
 
 SharePoint Migration API Retry Backoff in seconds.
 
-Change Conditions: ??
+Change Conditions: Retry backoff time can be changed. Can cause performance issues, not recommended.
 
 ### SharePoint Admin Url <a name="shareadurl"></a>
 {: .no_toc }
 Default Value: None
 
 The Url for the SharePoint admin center e.g. https://tenant-admin.sharepoint.com
 
-Change Conditions: ??
-
 ### Timeout <a name="stimeout"></a>
 {: .no_toc }
 Default Value: 1800000
 
 The timeout that will apply to communications with the SharePoint server.
 
-Change Conditions: ??
+Change Conditions: Timeout value (miliseconds) can be changed. Can cause performance issues, not recommended.
 
 ### Preserve File Created and Modified Dates <a name="persfilemoddate"></a>
 {: .no_toc }
 Default Value: On
 
 Preserve the Original File Created and Modified Dates.
 
-Change Conditions: ??
+Change Conditions: Disable to not preserve creation and modified dates of migrated files.
```

Comment:
If disabled the metadata of create and modified dates will not be migrated.

Reply:
done